source/Energinet.DataHub.MeteringPoints.Domain/SeedWork/Internals/EnumerationReflection.cs

```diff
@@ -40,7 +40,7 @@ public IEnumerable<EnumerationType> GetAll()
 
         public EnumerationType FromName(string name)
         {
-            if (_nameLookup.TryGetValue(name, out var type)) return type;
+            if (_nameLookup.TryGetValue(name.ToLower(), out var type)) return type;
```

Comment:
Use the invariant version of ToLower()

```suggestion
            if (_nameLookup.TryGetValue(name.ToLowerInvariant() out var type)) return type;
```